lib/locales/ru.js

```diff
@@ -0,0 +1,194 @@
+
+i18n.translations.ru = {
```

Comment:
this isn't the right place to do translations anymore and the latest version of master already has russian translations (see https://github.com/TelescopeJS/Telescope/blob/master/i18n/ru.i18n.json). Make sure you pull master and try again. Also, it's generally recommended to submit pull requests against `devel` instead of `master`